docker/generic/Dockerfile.kinetic

```diff
@@ -9,7 +9,7 @@ RUN apt-get update && apt-get install -y \
         libflann-dev \
         libgsl0-dev \
         libgoogle-perftools-dev \
-        libeigen3-dev
+        libeigen3-dev sudo gconf2
```

Comment:
Looks great. Just a minor thing, could you break this down into separate lines and sort them alphabetically?